packages/headless/src/controllers/commerce/field-suggestions/headless-field-suggestions-generator.ts

```diff
@@ -67,11 +65,8 @@ export function buildFieldSuggestionsGenerator(
   const controller = buildController(engine);
 
   const createFieldSuggestionsControllers = createSelector(
-    (state: CommerceEngineState) => state.fieldSuggestionsOrder!,
-    (state: CommerceEngineState) => state.commerceFacetSet,
-    (state: CommerceEngineState) => state.facetSearchSet,
-    (state: CommerceEngineState) => state.categoryFacetSearchSet,
-    (facetOrder, _commerceFacetSet, _facetSearchSet, _categoryFacetSearchSet) =>
```

Comment:
I reverted this change since the field suggestions controllers should **only** update when the field suggestions facet configuration changes, aka when the `fieldSuggestionsOrder` changes.